src/raftstore/store/util.rs

```diff
@@ -372,35 +373,13 @@ pub fn get_region_approximate_size(db: &DB, region: &metapb::Region) -> Result<u
     Ok(size)
 }
 
-/// Get the approximate number of records in the region.
-pub fn get_region_approximate_rows(db: &DB, region: &metapb::Region) -> Result<u64> {
+/// Get the approximate number of keys in write cf in the region.
+pub fn get_region_approximate_write_keys(db: &DB, region: &metapb::Region) -> Result<u64> {
```

Comment:
I think you can rename the function to `get_region_approximate_keys`. This function can get approximate keys in write cf and default cf, and return the larger one.
If you expose the concept of `write cf`, it will make things complicated.

Reply:
Here we should only consider the number of keys in write cf.  since we don't need to consider for RawKV and for other cases the number of keys in write cf will always bigger than the keys in default cf. And I think `key` has too many meanings in our system, we could not expose the concept of `write cf` here, but the in implementation of `SplitCheckObserver`, we still need to use the concept of `write cf`, then we will use `get_region_approximate_keys` together with the number of keys in `write cf`, I think that may make the code more complicated. Here one key in `write cf` indicate one version, maybe we could rename it to `get_region_approximate_versions`?